### PR TITLE
Update cloud-consolidation-managing-attributes.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: Skype/SfbHybrid/hybrid/cloud-consolidation-managing-attributes.md [hunk 1]
```diff
@@ -139,7 +139,7 @@ This option requires more effort and proper planning because users who were move
 7. Wait for user provisioning to complete. You can monitor user provisioning progress by running the following Teams PowerShell command. The following Teams PowerShell command returns an empty result as soon process is completed.
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/cloud-consolidation-managing-attributes.md:139]:
```suggestion
7. Wait for user provisioning to complete. You can monitor user provisioning progress by running the following Teams PowerShell command. The following Teams PowerShell command returns an empty result as soon as the process is completed.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, not related to this PR, but could you kindly also commit the additional suggested change? Many thanks.

**Comment by isrumnon** [comment]
@yogkumgit can we merge this?

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**